bad_ruby.rb

```diff
@@ -0,0 +1,5 @@
+class MyBadClass
+    def this_indentation
+      'is so wrong'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.